internal/examples/supervisor/supervisor/supervisor.go

```diff
@@ -437,16 +497,18 @@ func (s *Supervisor) runAgentProcess() {
 
 		case <-restartTimer.C:
 			s.startAgent()
+
+		case <-s.healthCheckTicker.C:
+			s.healthCheck()
 		}
 	}
 }
 
-func (s *Supervisor) applyConfigWithAgentRestart() {
+func (s *Supervisor) stopAgentApplyConfig() {
 	s.logger.Debugf("Restarting the agent with the new config.")
```

Comment:
If this no longer performs a restart, we should change the Debug statement.
```suggestion
	s.logger.Debugf("Stopping the agent to apply new config.")
```

Reply:
Fixed.